locale/de/translation.js

```diff
@@ -370,6 +370,7 @@ export const de_translation = {
     MN_STATUS_IS: 'Der Status der Masternode ist', //Your masternode status is
     MN_STATE: 'Die Masternode hat den Status <b>{state}</b>', //Your masternode is in <b>{state}</b> state
     MN_BAD_IP: 'Die IP-Adresse ist ungültig', //The IP address is invalid!
+    MN_BAD_PRIVKEY: 'The private key is invalid', //The private key is invalid
```

Comment:
```suggestion
    MN_BAD_PRIVKEY: '', //The private key is invalid
```
Should probably keep this empty